src/ocean/util/container/cache/model/Value.d

```diff
@@ -130,6 +130,18 @@ template Value ( size_t ValueSize )
             {
                 return &this.array;
             }
+
+            /*******************************************************************
+
+                Returns:
+                    length of underlying array
+
+            *******************************************************************/
+
+            public size_t length ( )
```

Comment:
Another option would be to have `empty` instead of `length` IIUC, of less general use though.